extension_methods/example/core_type_extensions.dart

```diff
@@ -32,6 +32,6 @@ extension Silliest on Object {
   bool notAgain(Object other) => this != other;
 }
 
-extension ExtremelySilly on Null {
+extension ExtremelySilly on void {
```

Comment:
Did a lint surface this? I think it's a bug in the lint.

Reply:
https://github.com/dart-lang/linter/issues/2680

Reply:
Thanks for filing the bug.

How about I just change this to `extension ExtremelySilly on Object?` for now? The file doesn't have any example with null safety, so would be good to add that, I think

Reply:
Done